### PR TITLE
Added Camera API Doc

### PR DESCRIPTION
**Pull request by tohman**

### CODE REVIEW

**Comment by tneil** [comment]
Is this change targetted for the delta or other branch?  We'll want to ensure that our upcoming API jsDocs are merged in their target release branch.

Master will need to stick with what is available today. .once we release this functionality it will merge down to master

**Comment by jeffheifetz** [comment]
I believe our current style would be to separate your commit into two separate files, blackberry_media and blackberry_media_camera.

File: api/blackberry_media_camera.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+/**
+* <div><p>
+*   Using Camera API allows to invoke the native camera application to record video or take a picture and receive a path to the file stored.
+*   The method are static and don't required to create an instance of a camera.
+* </p>
+* </div>
+* @toc {Media} Camera
+* @BB50+
+* @PB10
+* @namespace The Camera API provides the way to received path to files saved by native invoked camera application.
```
Comment (jeffheifetz) [inline api/blackberry_media_camera.js:32]: You currently have two different contributions to the Object description
- The text at the top of the comment tag implicitly is given an @description tag (This should be explicit)
- The text following the @namespace tag.

Please combine them into one

File: api/blackberry_media_camera.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+* @featureID blackberry.media.camera
+* @example
```
Comment (jeffheifetz) [inline api/blackberry_media_camera.js:34]: Please move this example to the takeVideo function

File: api/blackberry_media_camera.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+*   alert("Error occured");
+* } 
+*/
+blackberry.media.camera = {
+}
+
+/**
+* Opens the camera and return a path to the onComplete callback when a photo is taken.
+* @methodOf blackberry.media.camera
+* @param {onCompleteCallback} callback callback invoked when a picture is taken.
```
Comment (jeffheifetz) [inline api/blackberry_media_camera.js:61]: This is not the correct way to comment callback functions. Please replace these objects with lines more like the following.

```
@callback {function} onCompletion The onCompletion callback will be invoked when a picture is taken.
@callback {String} [onCompletion.filePath] The filesystem path of the photo or video taken by the camera.
```

**Pull request merged**